app/components/provider_interface/diversity_information_component.rb

```diff
@@ -10,7 +10,7 @@ def initialize(application_choice:, current_provider_user:)
     end
 
     def rows
-      return [] unless application_choice.application_form.equality_and_diversity_answers_provided?
+      return [] if application_choice.application_form.equality_and_diversity.blank?
```

Comment:
Do we need to add one spec on diversity_information_component_spec for this change?